### PR TITLE
docs: Added a note to not exclude http usage

### PR DESCRIPTION
**Pull request by johannes-b**
Signed-off-by: Johannes <johannes.braeuer@dynatrace.com>

### CODE REVIEW

File: documentation/project-setup.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Before continuing, please ensure a Keptn project is available with a valid Git u
 
 To function correctly, the dynatrace-service requires a Kubernetes secret containing the credentials for accessing the API of a Dynatrace tenant. The secret must have two components with the key names `DT_TENANT` and `DT_API_TOKEN`:
 
-* The `DT_TENANT` has to be set according to the appropriate [pattern](https://www.dynatrace.com/support/help/get-started/monitoring-environment/environment-id):
+* The `DT_TENANT` has to be set according to the appropriate [pattern](https://www.dynatrace.com/support/help/get-started/monitoring-environment/environment-id) shown below. While the following mentions the use of `https`, is supported to use `http` even though this is not recommended:
```
Comment (arthurpitman) [inline documentation/project-setup.md:16]: Perhaps "`http` is supported although this is not recommended"

Comment (johannes-b) [reply documentation/project-setup.md:16]: Thx, for the feedback. Addressed it accordingly.

**Review by j-poecher — APPROVED** [review]
lgtm

**Review by arthurpitman — APPROVED** [review]
LGTM

**Pull request merged**